.gitlab-ci.yml

```diff
@@ -58,7 +58,7 @@ variables:
         - logs_onboarding_host_container_auto_install/
         - logs_onboarding_container_auto_install/
 
-cross-components:
+cross-components: # do we need this?
```

Comment:
@cbeauchesne Do we need this job? I realized that we are launching this job nightly, but it is failing. Also we are launching "master" branch instead "main"

Reply:
No, it's some leftovers, you can remove it